development/compiling/compiling_for_android.rst

```diff
@@ -206,6 +206,53 @@ file in the ``bin\`` directory of your Godot source folder, so that the
 next time you build you will automatically have the custom templates
 referenced.
 
+Building the Godot Editor
+-------------------------
+
+Compiling the editor is done by calling SCons from the Godot
+root directory with the following arguments:
+
+-  Release Debug target (the editor will **only** compile with the **release debug** target.)
```

Comment:
I'd remove this notice as `target=debug` can also be used.

Reply:
I removed the notice as requested. You are correct that there are additional targets that the editor can be built on.